src/lib/defaultCommands.js

```diff
@@ -563,13 +563,14 @@ var defaultCmds = {
 
 	// START_COMMAND: Link
 	link: {
-		_dropDown: function (editor, caller, cb) {
+		_dropDown: function (editor, caller, selected, cb) {
```

Comment:
This causes an error in WYSIWYG mode as the call to `_dropDown` in `exec:` hasn't been updated to pass the `selected` parameter so `cb` is is being treated as `selected` and there is no `cb` passed.